.github/workflows/lint.yaml

```diff
@@ -0,0 +1,27 @@
+name: Lint
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - '*'
+  pull_request:
+
+jobs:
+  javascript:
+    name: JavaScript
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Node
+      uses: actions/setup-node@v2
+      with:
+        node-version: 12.x
+        cache: yarn
+    - name: Install dependencies
+      run: |
+        yarn install
+    - name: Lint
+      run: |
+        yarn lint --quiet
```

Comment:
Is there a specific reason to use `--quiet`?

Reply:
I copied this from the Dodona lint action, but I can omit this.